libs/application/templates/financial-aid/src/lib/utils.ts

```diff
@@ -79,6 +80,13 @@ export function findFamilyStatus(
 }
 
 export function hasActiveCurrentApplication(context: ApplicationContext) {
+  // On prod there should only be one active application per user
+  // When working with gervimaður we might need to have many active applications
+  const isProd = isRunningOnEnvironment('production')
+  if (!isProd) {
+    return false
+  }
```

Comment:
I am not sure but i think this is unneccassary, since we changed this feature i think 🤔 
So i think users can now apply more than once a month